test/__bootstrap__/expect.assertions.js

```diff
@@ -0,0 +1,40 @@
+//
+// Partially adapted from jasmine-expect-count
+// https://github.com/UziTech/jasmine-expect-count/blob/master/LICENSE
+//
+EXPECT_PATCHING_SCOPE: {
```

Comment:
☝️ Are you using a label and a block? ❤️

Reply:
I sure am!